src/Access/Common/AccessRightsElement.h

```diff
@@ -50,8 +55,11 @@ struct AccessRightsElement
 
     bool sameDatabaseAndTable(const AccessRightsElement & other) const
     {
-        return (database == other.database) && (any_database == other.any_database) && (table == other.table)
-            && (any_table == other.any_table);
+        return (database == other.database) && (any_database == other.any_database)
```

Comment:
Let's rename the function to `sameDatabaseAndTableAndParameter` and write a comment where it's used?

Reply:
done